packages/data-explorer/src/components/SearchComponent.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <div class="input-group mb-3">
+    <input
+      type="text"
+      v-model="searchText"
+      class="form-control"
+      placeholder="Search"
+      aria-label="Search data"
+      aria-describedby="mg-data-explorer-search"
+    />
+    <div class="input-group-append">
+      <button
```

Comment:
The rest of the interface (filters) react instantly. Having to push a button to search breaks my expectation. Maybe make search reactive?

Reply:
This is done by design, its search as opposed to typeahead ( filter style) , it's the same UX as the current data explorer, suggest to keep it for now and add it later as a feature if we think we want it ?